test/tests/viewer.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+/* global Viewer, describe, it, chai, afterEach */
+var expect = chai.expect;
+var viewer;
+var mockAPIs = {
+	reload: {
+		_action: 'reload'
+	},
+	updateId: {
+		_action: 'update',
+		id: 12345,
+		name: 'Dummy Test Viewer',
+		items: []
+	},
+	updateUrl: {
+		_action: 'update',
+		id: 12345,
+		name: 'Dummy Test Viewer',
+		items: [
+			{url: 'https://google.com'}
+		]
+	}
+};
+
+function mockAPI(action) {
+	viewer.socket.emit('echo', mockAPIs[action]);
+}
+
+describe('Connect to the server via io', function(){
+
+	it('Should connect via sockets on /screens', function(done){
+
+		viewer = new Viewer('http://localhost:3000');
+		viewer.socket.on('connect', function () {
+			expect(viewer.connectionState).to.be.true;
+			expect(viewer.data).to.not.be.undefined;
+			done();
+		});
+	});
+});
+
+describe('API', function () {
+
+	// remove all listeners
+	afterEach(function() {
+		viewer.removeAllListeners('change');
+		viewer.removeAllListeners('update');
+		viewer.removeAllListeners('reload');
+	});
+
+	it('Should reload when recieves reload', function (done) {
+		viewer.once('reload', function() { done() });
```

Comment:
Instead of wrapping `done` in an anonymous function, you could pass `done` directly as a value to `once`:
``` js
viewer.once('reload', done);
```